app/models/declaration/submissions/Submission.scala

```diff
@@ -41,12 +41,12 @@ case class Submission(
     } yield notificationSummaries.map(_.enhancedStatus)
   ).getOrElse(Seq.empty[EnhancedStatus])
 
-  lazy val blockAmendments: Boolean = latestDecId.isEmpty
+  lazy val blockOldAmendments: Boolean = latestDecId.isEmpty
```

Comment:
Just thinking the name `blockOldAmendments` might be confusing to our successors. Maybe leave as `blockAmendments` and add a comment explaining this `isEmpty` will only be true for legacy declarations created before this change?